website/content/en/docs/examples/gha.md

```diff
@@ -0,0 +1,90 @@
+---
+title: GitHub Actions
+weight: 10
+---
+
+## Running Lima on GitHub Actions
+
+Lima is also useful for:
+- Running commands on non-Ubuntu operating systems (e.g., Fedora for testing SELinux)
+- Emulating multiple hosts.
+
+These tasks can be also partially accomplished with containers like Docker,
+but they are still stick to Ubuntu's kernel configuration and cannot use non-Ubuntu features such as SELinux.
```

Comment:
I don't know what this means: "they are still stick to Ubuntu's kernel configuration"

Reply:
Reworded

```
These tasks can be also partially accomplished with containers like Docker.
However, those containers still have to run on the Ubuntu host's kernel, and cannot use features missing in Ubuntu, such as SELinux.

On the other hand, Lima executes virtual machines that does not use the Ubuntu host's kernel.
```

Reply:
Asked Copilot to improve it further
```
Lima is also useful for:
- Running commands on non-Ubuntu operating systems (e.g., Fedora for testing SELinux)
- Emulating multiple hosts

While these tasks can be partially accomplished with containers like Docker, those containers still rely on the Ubuntu host's kernel and cannot utilize features missing in Ubuntu, such as SELinux.

In contrast, Lima runs virtual machines that do not depend on the Ubuntu host's kernel.
```